tests/integration/wdio.conf.js

```diff
@@ -0,0 +1,273 @@
+    services: [
+      "firefox-profile", "selenium-standalone", ["image-comparison", {
+        baselineFolder: join(process.cwd(), "./tests/integration/tests/Visual_Baseline/"),
+        formatImageName: process.env.MOZ_HEADLESS ? "{tag}-headless-{width}x{height}" : "{tag}-{width}x{height}",
+        screenshotPath: join(process.cwd(), ".tmp/"),
+        savePerInstance: true,
+        autoSaveBaseline: true,
+      }],
+    ],
+
+    // Framework you want to run your specs with.
+    // The following are supported: Mocha, Jasmine, and Cucumber
+    // see also: https://webdriver.io/docs/frameworks.html
+    //
+    // Make sure you have the wdio adapter package for the specific framework installed
+    // before running any tests.
+    framework: "mocha",
+    //
+    // The number of times to retry the entire specfile when it fails as a whole
+    // specFileRetries: 1,
+    //
+    // Test reporter for stdout.
+    // The only one supported by default is 'dot'
+    // see also: https://webdriver.io/docs/dot-reporter.html
+    reporters: ["dot", "spec"],
+
+    //
+    // Options to be passed to Mocha.
+    // See the full list at http://mochajs.org/
+    mochaOpts: {
+        ui: "bdd",
+        timeout: 120000,
+    },
+    //
+    // =====
+    // Hooks
+    // =====
+    // WebdriverIO provides several hooks you can use to interfere with the test process in order to enhance
+    // it and to build services around it. You can either apply a single function or an array of
+    // methods to it. If one of them returns with a promise, WebdriverIO will wait until that promise got
+    // resolved to continue.
+    /**
+     * Gets executed once before all workers get launched.
+     * @param {Object} config wdio configuration object
+     * @param {Array.<Object>} capabilities list of capabilities details
+     */
+    // onPrepare: function (config, capabilities) {
+    // },
+    /**
+     * Gets executed just before initialising the webdriver session and test framework. It allows you
+     * to manipulate configurations depending on the capability or spec.
+     * @param {Object} config wdio configuration object
+     * @param {Array.<Object>} capabilities list of capabilities details
+     * @param {Array.<String>} specs List of spec file paths that are to be run
+     */
+    //beforeSession: function (config, capabilities, specs) {
+    //},
+    /**
+     * Gets executed before test execution begins. At this point you can access to all global
+     * variables like `browser`. It is the perfect place to define custom commands.
+     * @param {Array.<Object>} capabilities list of capabilities details
+     * @param {Array.<String>} specs List of spec file paths that are to be run
+     */
+    before: function (capabilities, specs) {
+      const chai = require("chai");
+
+      global.expect = chai.expect;
+      chai.Should();
+
+      global.baseUrl = "http://localhost:6060";
```

Comment:
Hhmm. this is defined here, and on L95 above:

```js
baseUrl: "http://localhost:6060",
```

Not sure if they are the same thing, or can be combined or extracted into a single variable that is shared between them.

Reply:
Not the same, I'll remove the wrong one.